README.md

```diff
@@ -430,7 +458,13 @@ func main() {
 	if err != nil {
 		log.Fatalf("Failed to describe index \"%s\". Error:%s", idx.Name, err)
 	} else {
-		fmt.Printf("Successfully found the \"%s\" index!\n", idx.Name)
+		desc := fmt.Sprintf("Description: \n  Name: %s\n  Dimension: %d\n  Host: %s\n  Metric: %s\n"+
+			"  DeletionProtection"+
+			": %s\n"+
+			"  Spec: %+v"+
+			"\n  Status: %+v\n",
```

Comment:
nit: Is this just to fix the printing of the pointer addresses? Not sure if I prefer the formatting to the simpler variety, but up to you.

Reply:
yeah, it's just some UX lipstick that was brought up in the bug bash. I agree it's sorta ugly, but the printed-out statement is really a lot better. I think we should keep?

Reply:
Sounds good.